lib/themes/dosomething/paraneue_dosomething/js/campaign/OrganDonation.js

```diff
@@ -1,84 +1,136 @@
 const $ = require('jquery');
+const forEach = require('lodash/collection/forEach');
+const template = require('lodash/string/template');
+
 import setting from '../utilities/Setting';
 import Takeover from 'takeover/Takeover';
 
+// Templates
+import organDonationTpl from 'campaign/templates/OrganDonation/takeover.tpl.html';
```

Comment:
Not needed for this PR, but I'd love to think about loading these from a `<script type="text/html">` on the action page so we don't need to include all this markup in our site-wide script bundle. (Or maybe starting to break things into different entry points so we only load all these scripts on pages where they're used.)